proof/crefine/RISCV64/SR_lemmas_C.thy

```diff
@@ -240,23 +240,6 @@ lemma cap_get_tag_ZombieCap:
             split: if_split_asm)
   done
 
-
-
-lemma cap_get_tag_ReplyCap:
-  assumes cr: "ccap_relation cap cap'"
-  shows "(cap_get_tag cap' = scast cap_reply_cap) =
-  (cap =
-      ReplyCap (ctcb_ptr_to_tcb_ptr (Ptr (cap_reply_cap_CL.capTCBPtr_CL (cap_reply_cap_lift cap'))))
-               (to_bool (capReplyMaster_CL (cap_reply_cap_lift cap')))
-               (to_bool (capReplyCanGrant_CL (cap_reply_cap_lift cap'))))"
```

Comment:
Out of curiosity, what has changed that we now don't need a lemma like this for the new ReplyCaps?

Reply:
Oh maybe I was too hasty in removing this. I'll check

Reply:
Yeah this shouldn't have been removed. I've now added something back and will include it in a push soon(ish)